File: practices/old_enough.py
# AB 7th Old Enough

age = int(input("What is your age?: "))

if age >= 18 and age < 100:
    print("You are old enough to vote!")
elif age >= 16 and age < 18:
    print("You are old enough to drive!")
elif age >= 15 and age < 16:
    print("You are old enough to get a learners permit!")
elif age >= 4 and age < 15:
    print("You should go to school!")

File: c_programming_2/practices/old_enough.c
// AB 7th Old Enough Practice

#include <stdio.h>

int main(){
    int age;

    printf("How old are you: ");
    scanf("%d", &age);

    if (age >= 18){
        printf("You are old enough to vote\n");
    }else if (age >= 16){
        printf("You aare old enough to drive\n");
    }else if (age >= 15){
        printf("You are old enough to get a learners permit\n");
    }else if (age >= 5){
        printf("You are old enough to go to school\n");
    }else {
        printf("You can't do anything :) hahaha...\n");
    }

    return 0;
}